### ISSUE TITLE
MHZ19B commands to set range not implemented

### ISSUE THREAD

**Issue by borpin** [esphome/issues#748]
Refers to Issue #709 
**Operating environment/Installation (Hass.io/Docker/pip/etc.):**

Hass.io / Docker
**ESP (ESP32/ESP8266, Board/Sonoff):**

WEMOS D1 Mini Rev 3.1.0
**Affected component:**

https://esphome.io/components/sensor/mhz19.html

**Description of problem:**
The commands to set the device range, as detailed in the most recent [datasheet ](https://www.winsen-sensor.com/d/files/MH-Z19B.pdf) for this device, have not been implemented.  The lack of an ability to set this configuration, **may** be affecting the values returned by the device.

https://github.com/esphome/esphome/blob/4310c14497be80eb47c33733910242b16aa7aa4a/esphome/components/mhz19/mhz19.cpp#L10-L15

![image](https://user-images.githubusercontent.com/1322749/67144119-35d3b480-f26b-11e9-9c06-c8c36b212e51.png)

![image](https://user-images.githubusercontent.com/1322749/67144123-497f1b00-f26b-11e9-8565-7b68ecb736bb.png)

**Problem-relevant YAML-configuration entries:**
```yaml
PASTE YAML FILE HERE

```

**Logs (if applicable):**

```
PASTE DEBUG LOG HERE

```

**Additional information and things you've tried:**

**Comment by OttoWinter** [comment]
> The lack of an ability to set this configuration, may be affecting the values returned by the device.

True, but as far as I see the detection range only affects what range of values the device can send. It doesn't make the sensor return "wrong" values.

I will move this to the feature-requests repo because it's a feature request.
